content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Hello Minikube
```

Comment:
I'd localize this title. Most localizations do, I think.

Reply:
Yup missed it, Done now : )